ui/apps/platform/src/types/telemetryConfigService.proto.ts

```diff
@@ -0,0 +1,5 @@
+export type TelemetryConfig = {
+    userId: string;
+    endpoint: string;
+    storageKeyV1: string;
+};
```

Comment:
Not a blocker to merge: maybe just as well to move the type into services/TelemetryConfigService.ts file, because the type is in a special place
https://github.com/stackrox/stackrox/commit/8be5abb209ef162ef5387d3908742775e7117e6e

different from the usual backend proto/storage folder, which has an unrelated type left over from the original telemetry service:
https://github.com/stackrox/stackrox/blob/master/proto/storage/telemetry.proto